components/server/src/ome/security/basic/OmeroInterceptor.java

```diff
@@ -91,6 +105,11 @@
 
     private static Logger log = LoggerFactory.getLogger(OmeroInterceptor.class);
 
+    /** thread-safe and may be removed without warning;
+     *  used by {@link omero.cmd.graphs.BaseGraphTraversalProcessor#deleteInstances(String, Collection)} */
+    public static SetMultimap<Session, Long> DELETED_GROUP_LINK_IDS =
+            Multimaps.synchronizedSetMultimap(HashMultimap.<Session, Long>create());
```

Comment:
I know we're still in the "finding a solution" phase, but a global synchronized map worries me.